dvc/output/base.py

```diff
@@ -31,7 +31,11 @@ def __init__(self, path):
 
 class OutputAlreadyTrackedError(DvcException):
     def __init__(self, path):
-        msg = f"output '{path}' is already tracked by SCM (e.g. Git)"
+        msg = f""" output '{path}' is already tracked by SCM (e.g. Git).
+    You can remove it from git, then add to DVC.
+        To stop tracking from git:
```

Comment:
sorry that it's late a bit, but it should be probably`git` -> `Git` like it is in the previous line

Reply:
Fixed in https://github.com/iterative/dvc/commit/7672e324d221198a307b5b7963ff103bad2b92ab . Thanks @shcheklein ! :pray: